src/Common/Crypto/CryptoGen.php

```diff
@@ -28,9 +28,9 @@
 
 namespace OpenEMR\Common\Crypto;
 
+use Exception;
 use OpenEMR\Common\Utils\RandomGenUtils;
 
-#[\AllowDynamicProperties]
```

Comment:
Did you mean to remove this?

Reply:
No but this is strange as I saw this before I pushed and copied corrected version to local and pushed. I then verified but both versions were present in master so beats me and will correct.
Dynamic properties were deprecated and this tag is for php8.2 but errors silently on lower version.
I'll try to track down and resolve the properties so we don't have to worry in future.